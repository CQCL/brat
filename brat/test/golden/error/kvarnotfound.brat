f :: { Bit -o Bit }
f = { x => y }
