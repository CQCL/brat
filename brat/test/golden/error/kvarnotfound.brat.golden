Error in test/golden/error/kvarnotfound.brat@2:12
f = { x => y }
           ^

  y not found in kernel context

